src/citrine/resources/data_concepts.py

```diff
@@ -27,7 +29,23 @@
 CITRINE_TAG_PREFIX = 'citr_auto'
 
 
-class DataConcepts(DictSerializable, PolymorphicSerializable['DataConcepts'], ABC):
+class DataConceptsMeta(DictSerializableMeta):
+    """Data Concepts metaclass for handling serialization."""
+
+    def __init__(cls, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        resolved = next((b.typ for b in cls.__bases__ if getattr(b, "typ", None) is not None),
+                        None)
```

Comment:
Once we can import a gemd-python version that has abstract + concrete classes, this can be simplified.  That is expected to be released next cycle.